src/main/core/support/units.rs

```diff
@@ -2,13 +2,17 @@
 Types for parsing/deserializing unit values.
 
 ```
+# use shadow_rs::core::support::units::*;
+# use std::str::FromStr;
+# fn main() {
```

Comment:
I don't think you need the `fn main()` ?

Reply:
Oops, fixed!